group-coordinator/src/test/java/org/apache/kafka/coordinator/group/assignor/GeneralUniformAssignmentBuilderTest.java

```diff
@@ -49,6 +50,7 @@ public class GeneralUniformAssignmentBuilderTest {
     private final String memberA = "A";
     private final String memberB = "B";
     private final String memberC = "C";
+    private final ConsumerGroupSubscriptionModel groupSubscriptionModel = HETEROGENEOUS;
```

Comment:
nit: As we use the same value everywhere, I would directly inline `HETEROGENEOUS`.